chia/full_node/full_node_api.py

```diff
@@ -61,6 +60,11 @@
 from chia.util.limited_semaphore import LimitedSemaphoreFullError
 from chia.util.merkle_set import MerkleSet
 
+if TYPE_CHECKING:
+    from chia.full_node.full_node import FullNode
```

Comment:
would you mind leaving a comment explaining what this is for?
Is it to avoid cyclical dependencies, or just to keep unnecessary imports out of production?

Reply:
I would suggest that this should be a matter of general practice and not a thing that is commented each place.  I generally only do this for known cyclic imports.  Could make a pr to the pretty good practices doc if we want to 'formalize' this.

Reply:
mypy doesn't mind circular dependencies then I take it?

Reply:
Nope.